.github/workflows/end_to_end_test.yaml

```diff
@@ -12,8 +12,14 @@ jobs:
     steps:
       - name: Echo hello world
         run: echo "hello world"
+      - name: File permission for /usr/local/bin
+        run: ls -ld /usr/local/bin | grep drwxrwxrwx
       - name: Proxy set in /etc/environment
-        run: cat /etc/environment | grep PROXY
+        run: cat /etc/environment | grep HTTP_PROXY
```

Comment:
That wouldn't catch HTTPS_PROXY not being set, no? Do you know whether we rely on it being set, so perhaps add an additional check for it?

Reply:
The "Install microk8s" step would be testing the HTTPS_PROXY and NO_PROXY settings.
I added a comment for it.

This line would be testing whether the write file succeed. Since all the proxy setting are written into the file in one operation.